components/about-us.js

```diff
@@ -0,0 +1,55 @@
+import portfolio_content from "../portfolio_content.json";
```

Comment:
Use absolute paths instead of relative paths.
```suggestion
import portfolio_content from "@/portfolio_content.json";
```

Or if you have moved it already
```suggestion
import portfolio_content from "@/components/portfolio_content.json";
```